### PR TITLE
Docs preview

### PR DESCRIPTION
**Pull request by pngwn**
This PR will build the local docs and serve them on port `3456` of localhost. This will allow us to validate changes when working on the docs locally.

There will be a partner PR in the sites repo shortly.

### CODE REVIEW

**Comment by pngwn** [comment]
Ideally, we should publish the CLI to npm rather than installing via git but this will work for now.

File: package.json [hunk 1]
```diff
@@ -39,5 +40,8 @@
 		"rollup": "^2.55.0",
 		"typescript": "^4.3.5"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"action-deploy-docs": "github:sveltejs/action-deploy-docs#main"
```
Comment (benmccann) [inline package.json:45]: should this be in `devDependencies` instead?

**Comment by benmccann** [comment]
Nice!! This will be great to have :smile:

**Comment by benmccann** [comment]
Could you add something to the `README.md` about how to use this?

File: package.json (hunk 1 above)
Comment (pngwn) [reply package.json:45]: yip

**Comment by pngwn** [comment]
Should we put it in both this and the sites repo? I'm not sure what the standard entry-point will be for modifying docs.

**Comment by benmccann** [comment]
Most docs / sites PRs come in via this repo because they're modifying stuff in https://github.com/sveltejs/kit/tree/master/documentation. You could put it just in the sites repo and then point this one at it though so that we don't have to duplicate. Usually you can get away with modifying the docs without needing to run them, but sometimes it's not clear how changes will show up

**Comment by pngwn** [comment]
Documented this here: https://github.com/sveltejs/sites/blob/master/sites/kit.svelte.dev/README.md#previewing-local-docs-changes and linked toi it from the `kit` root readme.

File: README.md [hunk 2]
```diff
@@ -79,3 +79,9 @@ To run a single integration test, provide the `FILTER` env var with the test nam
 You can run the test server with `cd packages/kit/test/apps/basics; pnpm run dev` to hit it with your browser.
 
 You may need to install some dependencies first e.g. with `npx playwright install-deps` (which only works on Ubuntu).
+
+### Documentation
+
+All documentation for SvelteKit is in the `documentation` directory, any improvements should be made as a Pull Request to this repository. The documentation is served via and API, the site itself is located in the [`sites` repository](https://github.com/sveltejs/sites).
+
+If you wish toi preview documentation changes locally, please follow the instructions here: [Previewing local docs changes](https://github.com/sveltejs/sites/blob/master/sites/kit.svelte.dev/README.md#previewing-local-docs-changes).
```
Comment (benmccann) [inline README.md:87]: typo
```suggestion
If you wish to preview documentation changes locally, please follow the instructions here: [Previewing local docs changes](https://github.com/sveltejs/sites/blob/master/sites/kit.svelte.dev/README.md#previewing-local-docs-changes).
```

**Review by benmccann — APPROVED** [review]
lgtm. thanks again for this!

**Pull request merged**